akka-actor-typed/src/main/scala/akka/actor/typed/Behavior.scala

```diff
@@ -412,9 +452,16 @@ object Behavior {
         throw new IllegalArgumentException(s"cannot execute with [$behavior] as behavior")
       case d: DeferredBehavior[_] ⇒ throw new IllegalArgumentException(s"deferred [$d] should not be passed to interpreter")
       case IgnoreBehavior         ⇒ Behavior.same[T]
-      case s: StoppedBehavior[T]  ⇒ s
-      case f: FailedBehavior      ⇒ f
-      case EmptyBehavior          ⇒ Behavior.unhandled[T]
+      case u: UnstashingBehavior[T] ⇒
+        msg match {
+          case s: Signal ⇒
+            u.receiveSignal(ctx, s)
+          case _ ⇒
+            throw new InvalidMessageException(s"Unstashing behavior should never receive messages but got [$msg]")
```

Comment:
the scala doc says, `InvalidMessageException is thrown when an invalid message is sent to an Actor; Currently only `null` is an invalid `message.`

Is that the best exception to throw? I think I saw one option which now of course I can not find.

Reply:
`IllegalState` could be better. It will really never happen once this PR is done.

Reply:
That exc may make more sense